development/cpp/configuring_an_ide/android_studio.rst

```diff
@@ -4,15 +4,15 @@ Android Studio
 ==============
 
 `Android Studio <https://developer.android.com/studio>`_ is a free
-`JetBrains <https://www.jetbrains.com/>`_ IDE for Android development.
-It has a feature-rich editor which supports Java and C/C++. It can be used to
+IDE for Android development made by google and `JetBrains <https://www.jetbrains.com/>`_,
```

Comment:
```suggestion
IDE for Android development made by `Google <https://about.google/>`_ and `JetBrains <https://www.jetbrains.com/>`_.
```

Proper capitalization and we can still link to Google (we do link to Microsoft in other articles).